appeals/web/src/server/lib/error-handlers/change-screen-error-handlers.js

```diff
@@ -39,6 +39,14 @@ export const errorLastName = (/** @type {Error}*/ errors) => {
 		: undefined;
 };
 
+export const errorOrgNameAllowEmpty = (/** @type {Error}*/ errors) => {
```

Comment:
Again super pedantic but in general it's better to avoid abbreviations where possible